spec/std_ext_spec.rb

```diff
@@ -6,6 +6,7 @@
   end
 
   it "overrides parse" do
+    expect(Date.parse()).to eq Date.new(-4712, 1, 1)
```

Comment:
I'd like to check that Date.parse() returns same result of original.
Please change to compare with `Date._wareki_parse_orig()` instead of -4712-01-01.

Reply:
Makes sense.  Done!